docs/book_files/conrefs.html.yml

```diff
@@ -0,0 +1,2 @@
+storageclass: |
+  [Creating a StorageClass](../configuration/creating_volumestorageclass.md)
```

Comment:
it seems a newline is missing in the end of the file (same for the pdf version)
![image](https://user-images.githubusercontent.com/57443811/145705781-a5ef375e-5ec3-49cd-96ec-591633ee4948.png)

Reply:
this file is being used for testing conrefs. i'll delete if not used.

Reply:
thanks